learning/features/shading/shading_language.rst

```diff
@@ -377,41 +377,45 @@ CanvasItem (2D) - VertexShader
 +-----------------------------------+--------------------------------------------------------------------------------------------+
 | const float *TIME*                | Time (in seconds)                                                                          |
 +-----------------------------------+--------------------------------------------------------------------------------------------+
+| const bool *AT_LIGHT_PASS*        | Whether the shader is being run for a lighting pass (happens per affecting light)          |
++-----------------------------------+--------------------------------------------------------------------------------------------+
 
 CanvasItem (2D) - FragmentShader
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-+-------------------------------------+------------------------------------------------------------------------------+
```

Comment:
Yeah but Sphinx is designed around reST, I don't particularly want to change markup language yet another time (we already went MarkDown -> Textile -> reST). reST tables are a bit cumbersome, but the rest is pretty good for documentation IMO.